data_dictionary/data_structures_extra/costing.json

```diff
@@ -162,10 +162,15 @@
  },
  "costing.levelized_CoE": {
   "data_type": "FLT_0D",
-  "documentation": "Levelized cost of electiricity (total cost / total electricy generated)",
+  "documentation": "Levelized cost of electiricity (total cost / total electricity generated)",
   "type": "static",
   "units": "$/kWh"
  },
+ "costing.model": {
```

Comment:
@adrianaghiozzi if you want to add the model name, we should try to conform to what IMAS generally does for all of its IDSs. I'll add some logic to `generate_dd` to copy the `.code` tree under the IDSs that we have added. This should allow you to use `costing.code.name`.

Reply:
@orso82 Thanks, that would be great - this is just needed for the risk actor because the costs are grouped differently depending on whether it's ARIES or Sheffield (costing each coil system separately vs. all together) so the model determines how many fields are needed in dd.risk.engineering.loss

Reply:
Uhm... I have not taken a look at the code, but it sounds like you are hardcoding the number of fields based on the model, which is definitely not a robust thing to do. Perhaps it would be best to just pick a model (say ARIES), and use the TF costs from that model for the evaluation of risk.